leakcanary-android-core/src/main/res/values/leak_canary_strings.xml

```diff
@@ -23,8 +23,13 @@
   and open sourced by <a href="https://twitter.com/Piwai">Pierre-Yves Ricau</a> at <a href="https://square.github.io">Square</a>.<br><br>
   You can learn more about memory leaks at <a href="https://squ.re/leaks">https://squ.re/leaks</a>.<br><br>
   We welcome contributions from the community - please do not hesitate to
-  <a href="https://github.com/square/leakcanary/issues">report an issue</a> or open a pull request!<br>
+  <a href="https://github.com/square/leakcanary/issues">report an issue</a> or open a pull request!<br><br>
+  Heap dump is currently <i>%3$s</i>.<br>
 ]]></string>
+  <string name="leak_canary_heap_dump_disabled">disabled: %s</string>
+  <string name="leak_canary_heap_dump_enabled">enabled</string>
+  <string name="leak_canary_heap_dump_disabled_config_changed">config changed via code</string>
+  <string name="leak_canary_heap_dump_disabled_build_non_debuggable">build is non debuggable</string>
```

Comment:
I think these error messages can be more descriptive. Something like:
- Dumping heap disabled via LeakCanary.Config
- Debugger is attached and dumping heap during debugging is disabled

I'm not great with descriptions though, @pyricau what do you think?

Reply:
Indeed. Please check again and let me know if these are better. I tried to be concise without being too much cryptic.